test/Microsoft.AspNetCore.Razor.Test.Common/Langauge/IntegrationTests/IntegrationTestBase.cs

```diff
@@ -184,9 +184,6 @@ protected void AssertDesignTimeDocumentMatchBaseline(RazorCodeDocument document)
             Assert.NotNull(syntaxTree);
             Assert.True(syntaxTree.Options.DesignTimeMode);
```

Comment:
Can't pass just the mappings. Need the source as well so we can pull from the generated code and from the cshtml file to compare mappings.